content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -974,6 +974,13 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `LegacyNodeRoleBehavior`: When disabled, legacy behavior in service load balancers and
   node disruption will ignore the `node-role.kubernetes.io/master` label in favor of the
   feature-specific labels provided by `NodeDisruptionExclusion` and `ServiceNodeExclusion`.
+- `KubeletTracing`: Add support for distributed tracing in the kubelet.
+  When enabled, kubelet CRI interface and authenticated http servers are instrumented to generate
+  OpenTelemetry trace spans. Sampling rate and collector address are configurable. When enabled without
+  a tracing configuration, a parent span's sampling decision is respected and the default collector
+  address of 0.0.0.0:4317 is set. See
+  [KubeletTracing](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/2831-kubelet-tracing/README.md)
+  for more details.
```

Comment:
Instead of linking to a KEP, please write a documentation page and link to that.

I would add it within https://kubernetes.io/docs/reference/node/ - for example, a new page at https://kubernetes.io/docs/reference/node/telemetry/

Reply:
ok for this PR I'll remove the link to the KEP, and I'll add a docs/reference/node/telemetry/. page in a follow-up, so this PR can clear.

Reply:
all set - added to the `cluster administration, system-tracing` docs that were added w/ apiserver tracing